benchmarks/datasets/join/create_index/pg_search.sql

```diff
@@ -0,0 +1,75 @@
+CREATE INDEX pages_index ON pages
+USING bm25 ("id",
+    "content",
+    "title",
+    "parents",
+    "fileId",
+    "sizeInBytes",
+    "createdAt"
+)
+WITH (
+    key_field = 'id',
```

Comment:
Let's add all fields as fast fields.
```suggestion
    key_field = 'id',
    numeric_fields = '{
        "sizeInBytes": {
            "fast": true
        },
        "createdAt": {
            "fast": true
        }
    }',
```

Reply:
Numeric fields are fast by default.